application/config/database.php

```diff
@@ -53,7 +53,7 @@
 	'persistent'    => FALSE,
 	'connection'    => array
 	(
-		'type'     => 'mysql',
+		'type'     => 'mysqli',
```

Comment:
We should leave this to be mysql since nacoma is using PDO it cannot work with mysql. It only uses it for database information at the moment.